### PR TITLE
Added -allow-stale option into deploy

### PR DESCRIPTION
**Pull request by bogdanov1609**
Hey! 
I tried adding an option allow-stale for deploy command for simple reasons: 
We have a pretty bad network between servers and sometimes, when we apply deploy, we get errors because nomad didn't have time to replicate the data about the deployment to neighboring servers.

Would be happy to receive your feedback!

### CODE REVIEW

**Comment by jrasell** [comment]
thanks for this @bogdanov1609 and apologies I have not got round to looking sooner. there are a couple of merge conflicts, are you able to resolve these along with my commented review?

File: command/deploy.go [hunk 1]
```diff
@@ -68,6 +68,9 @@ General Options:
   -var-file=<file>
     Used in conjunction with the -job-file will deploy a templated job to your
     Nomad cluster. [default: levant.(yaml|yml|tf)]
+
+  -allow-stale
```
Comment (jrasell) [inline command/deploy.go]: pedantic but could we move this up and keep the flags list in alphabetical order?

File: levant/structs/config.go [hunk 2]
```diff
@@ -43,4 +43,7 @@ type Config struct {
 	// VaiableFile contains the variables which will be substituted into the
 	// templateFile before deployment.
 	VaiableFile string
+
+	// AllowStale sets consistency level for nomad query - https://www.nomadproject.io/api/index.html#consistency-modes
+	AllowStale bool
```
Comment (jrasell) [inline levant/structs/config.go]: pedantic but could we move this up and keep the flags list in alphabetical order?

**Comment by bogdanov1609** [comment]
Thanks for the feedback! I've updated the PR to resolve conflicts.

**Pull request merged**